wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -115,6 +134,32 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
         handleIntent(intent!!)
     }
 
+    override fun onWalletFragmentViewCreated() {
+        AppDatabase.getAppDatabase().blockchainStateDao().load().observe(this, Observer {
+            blockchainState = it
+            walletFragment.setBlockchainState(blockchainState)
+        })
+        dashPayViewModel = ViewModelProvider(this)[DashPayViewModel::class.java]
+        dashPayViewModel!!.isPlatformAvailableLiveData.observe(this, Observer<Resource<Boolean?>?> { status ->
+            if (status?.status === Status.SUCCESS) {
+                if (status.data != null) {
+                    isPlatformAvailable = status.data
+                }
+            } else {
+                isPlatformAvailable = false
+            }
+            walletFragment.setPlatformAvailability(isPlatformAvailable)
+        })
+        AppDatabase.getAppDatabase().blockchainIdentityDataDao().loadBase().observe(this,
```

Comment:
`blockchainIdentityDataDao` was renamed `blockchainIdentityDataDaoAsync`.  This PR must be based on an older commit from the `evonet-develop` branch.

Reply:
`blockchainIdentityDataDao()` is now only to be used in `suspend` methods, while `blockchainIdentityDataDaoAsync()` is actually asynchronous.

Reply:
Fixed on https://github.com/sambarboza/dash-wallet/commit/964b74e840cd75422b5beaaee8e6c853f207c2c8